Source/Atomic/Input/Input.cpp

```diff
@@ -1510,6 +1595,13 @@ void Input::Initialize()
 
     graphics_ = graphics;
 
+// ATOMIC BEGIN
+    //  get user gamecontroller configs for SDL
+    FileSystem* fs = GetSubsystem<FileSystem>();
+    const String configName = fs->GetUserDocumentsDir() + "/gamecontrollerdb.txt"; 
+    SDL_GameControllerAddMappingsFromFile(configName.CString());
+// ATOMIC END

```

Comment:
Should we file a followup issue to include the community controller database (https://raw.githubusercontent.com/gabomdq/SDL_GameControllerDB/master/gamecontrollerdb.txt) into the AtomicEditor data files, so it is guaranteed to be available and can be copied to deployed applications?